packages/titanium-docgen/generators/typescript_generator.js

```diff
@@ -1068,10 +1065,10 @@ class NamespaceNode extends MemberNode {
 			this.parseMethods(moduleDoc.methods);
 		}
 		if (this.interfaces.length) {
-			this.interfaces.forEach(node => this.findDuplicates(node.name));
+			this.interfaces.forEach(node => this.findDuplicates(node));
```

Comment:
Since this is allowed by TypeScript, isn't this check kind of pointless for interfaces now? Inside `findDuplicates` you return early if the input node is an instance of `InterfaceNode` anyway.

Reply:
Yes, you're right.
Fixed.